device-types/Cisco/C1000-16FP-2G-L.yaml

```diff
@@ -3,52 +3,86 @@ manufacturer: Cisco
 model: C1000-16FP-2G-L
 slug: c1000-16fp-2g-l
 part_number: C1000-16FP-2G-L
-u_height: 1
 comments: '[Cisco Catalyst 1000 Series Switches Data Sheet](https://www.cisco.com/c/en/us/products/collateral/switches/catalyst-1000-series-switches/nb-06-cat1k-ser-switch-ds-cte-en.html)'
+u_height: 1
 is_full_depth: false
+console-ports:
+  - name: con0
+    type: rj-45
+  - name: USB con0
+    type: usb-mini-b
+power-ports:
+  - name: Built-in
```

Comment:
Current documentation addresses the PSUs as PSU1, PSU2, etc. Please revert this to PS-1 or PSU1 (my preference is PSU1), or provide documentation where it is called Build-in. Thanks!